.github/workflows/comment_on_first_contribution.yml

```diff
@@ -0,0 +1,56 @@
+name: Comment on first contribution
+
+on:
+  workflow_call:
+    inputs:
+      pull_request_message:
+        description: >
+          The Markdown message to comment when a contributor creates their first
+          pull request.
+        type: string
+        required: false
+      issue_message:
+        description: >
+          The Markdown message to comment when a contributor creates their first
+          issue.
+        type: string
+        required: false
+
+jobs:
+  comment:
+    if: >
+      (
+        (github.event_name == 'pull_request_target' &&
+         inputs.pull_request_message) ||
+        (github.event_name == 'issues' && inputs.issue_message)
+      ) &&
+      github.event.action == 'opened' &&
+      contains(
+        fromJSON('["FIRST_TIMER", "FIRST_TIME_CONTRIBUTOR"]'),
+        github.event_name == 'issues'
+          && github.event.issue.author_association
+          || github.event.pull_request.author_association
+      )
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
I don’t think this is necessary. According to the GitHub documentation, workflows from the `pull_request_target` already have write access.

Reply:
But this can be triggered by issues too.

Reply:
Oh, maybe you meant that we can rid of the `pull-requests` key. That makes sense 👍